src/main/java/javax/jmdns/impl/DNSCache.java

```diff
@@ -218,10 +218,10 @@ public boolean addDNSEntry(DNSEntry dnsEntry) {
             if (entryList == null) {
                 entryList = new ArrayList<>(3);
                 entryList.add(dnsEntry);
```

Comment:
I assume you also wanted to delete this line?

Reply:
Sorry Kai, which line you are referring to?

Reply:
The one where I placed the comment: line 220. It looks redundant, since you've added line 222.

Reply:
Yes, please remove. Good catch, thanks!